docs/DeveloperGuide.adoc

```diff
@@ -286,9 +286,50 @@ Refer to the guide <<DevOps#, here>>.
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="75%",cols="<15%,<25%,<30%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
+|Priority |As a ... |I want to ... |So that ...
+|`* * *` |beginner cook |find new recipes easily |I don't waste time searching though recipes from different sources
+
+|`* * *` |regular cook |record my own recipes |I can refer to them easily in future
+
+|`* * *` |forgetful person |add ingredients for my planned meals to a grocery list easily |I know what I need to get when shopping
+
+|`* * *` |disorganized person |keep track of the ingredients I have at home |plan my meals better
+
+|`* * *` |busy student |cook a meal with the ingredients I already have |I don't waste time on grocery shopping
+
+|`* * *` |low-income individual |cook a meal with the ingredients I already have |I can save money
+
+|`* * *` |person with seafood allergy |cook meals that don't contain seafood |I do not have an allergic reaction
+
+|`* *` |cooking enthusiast |edit recipes |I can tweak a recipe to my liking
+
+|`* *` |regular cook |set a timer during meal preparation |I can control the quality of my meal
+
+|`* *` |CS student |cook a quick meal |spend more doing CS2103T
+
+|`* *` |vegetarian |find recipes that don't contain meat |I can keep to my diet constraints
+
+|`* *` |picky eater |choose recipes that only contain the food I like |I can enjoy the meals I cook
+
+|`* *` |working adult |plan meals for the next week |I can buy all the ingredients I need in one trip
+
+|`* *` |person with health issues |record the meals I eat |I can share the information with my doctor easily
+
+|`*` |health-conscious person |keep track of the nutritional value of the food I eat |I can meet my nutritional goals
+
+|`*` |regular gym-goer |keep track of my dietary intake |I can meet my fitness goals
+
+|`*` |obesity fighter |keep track of my calorie and fat intake |I can lose weight
+
+|`*` |stay-at-home parent |plan a variety of meals for the week |I can make sure that my family eats healthily
+
+|`*` |kiasu parent |know how much ingredients I need for 2 weeks |my family can survive an epidemic
```

Comment:
... 2 weeks |My family can survive an epidemic

Reply:
I'm starting with small letters, just that the rest start with "I"